index.js

```diff
@@ -152,30 +192,15 @@ function sendNotification(endpoint, params) {
 
       if (vapid && !isGCM) {
         // VAPID isn't supported by GCM.
+        vapid.audience = urlParts.protocol + '//' + urlParts.hostname;
```

Comment:
Also filed https://github.com/webpush-wg/webpush-vapid/issues/23 so that the spec gets clarified. I agree with @marco-c that it's not quite clear what the value or intent is in the spec. 

Well, that and apparently I failed to be properly caffeinated or focused when I read that the first time.